src/docs/howto_contribute.md

```diff
@@ -149,112 +141,60 @@ run relevant tests. If you're not sure what those are,
+Note that while only committers are available to add in the Assignees field,
+any pants contributors may still post reviews if you provide them with a link
+manually or they see it in the google group or github notifications.
 
 #### Iterating
 
-If reviewers have feedback, there might be a few iterations before
-finally getting a Ship It. As reviewers enter feedback, the rbcommons
-page updates; it should also send you mail (but sometimes its "spoof"
-fails).
+If reviewers post any feedback
+([for more information on providing feedback see](https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/)),
+there might be a few iterations before finally getting a Ship It. As reviewers enter
+feedback, the github page updates; it should also send
+you mail as long as you are `Subscribed` to notifications for the pull request.
 
 If those reviews inspire you to change some code, great. Change some
-code, commit locally. To update the code review with the new diff where
-`<RB_ID>` is a review number like `123`:
+code and commit locally. When you're ready to update the pull request with your changes,
+push to the relevant branch on your fork as you did before:
 
     :::bash
-    $ ./rbt post -o -r <RB_ID>
-
-Look over the fields in the web form; perhaps some could use updating.
-Press the web form's Publish button.
+    $ git push <your-username> $FEATURE_BRANCH
 
-If need a reminder of your review number, you can get a quick list with:
-
-    :::bash
-    $ ./rbt status
-    r/1234 - Make pants go even faster
+Look over the fields in the pull request you created earlier; perhaps some could use updating.
+Press the web form's `edit` button.
+    
+If at any point you need to make changes that will fundamentally overhaul a review,
+consider temporarily removing the `reviewable` label in order to let reviewers know
+to hold off until the code is ready.
 
 ### Commit Your Change
 
-At this point you've made a change, had it reviewed and are ready to
-complete things by getting your change in master. (If you're not a
+At this point you've made a change, had it reviewed (and received one or more Ship Its!) and
+are ready to complete things by getting your change in master. (If you're not a
 committer, please ask one to do this section for you.)
 
-    :::bash
-    $ cd /path/to/pants/repo
-    $ ./build-support/bin/ci.sh
-    $ git checkout master
-    $ git pull
-    $ ./rbt patch -c <RB_ID>
-
-Here, ensure that the commit message generated from the review summary
-is accurate, and that the resulting commit contains the changes you
-expect. (If `rbt` gives mysterious errors, pass `--debug` for more info.
-If that doesn't clarify the problem, mail pants-devel (and include that
-`--debug` output).)
-
-Finally,
-
-    :::bash
-    $ git push origin master
+A committer should push the `Squash and merge` button on the PR, and ensure that the
+commit message generated from the review summary is accurate. In particular, the title should
```

Comment:
Noting that a small defeat to the drive towards fully hosted infra, we can write a check webhook that we force to be green for a commit to be merged.  That we could run on Amazon and it could check commit messages, etc to our content. We could pair that with a PR creation script as well as a merge script to make this easier-ish to get right from the reviewee and reviewer ends. Lots of "we" in there and I'd be happy to be the we who gets this all setup if folks agree on what to check, etc.